docs/overrides/partials/integrations/analytics/custom.html

```diff
@@ -0,0 +1,93 @@
+<script src="https://cdn.amplitude.com/libs/analytics-browser-1.9.1-min.js.gz"></script>
+<script type="text/javascript">
+// Difficult to get the value from mkdocs-material config to be interpolated as
+// a boolean, so we're using a string and converting it to a boolean here.
+const amplitudeEnabledParam = "{{ config.extra.analytics.enabled }}"
+const amplitudeEnabled = amplitudeEnabledParam === 'enabled'
+const amplitudeKey = "{{ config.extra.analytics.property }}"
+const amplitudeUrl = "{{ config.extra.analytics.url }}"
+
+
+const addUrl = (event) => {
+  const deviceId = amplitude.getDeviceId()
+  const { href = '' } = event.target
+  const url = new URL(href)
+  url.searchParams.append('deviceId', deviceId)
+  event.target.href = url.toString()
+}
+
+const removeUrl = (event) => {
+  const { href = '' } = event.target
+  const url = new URL(href)
+  url.searchParams.delete('deviceId')
+  event.target.href = url.toString()
+}
+
+const urls = [
+  'https://app.prefect.cloud',
+  'https://app.stg.prefect.dev',
+  'https://app.prefect.dev',
+  'http://app.prefect.cloud',
```

Comment:
fyi, it's not possible to load any of these sites over http, we have HSTS enabled. I would suggest removing these, as it's a bug if we ever use non-HTTPS URLs.

Reply:
There is at least one http url currently in the docs and since we have open-source contributions I figured it would be easier to cover the edge-case here